packages/grafana-ui/src/components/index.ts

```diff
@@ -235,6 +235,9 @@ export { SelectMenuOptions } from './Select/SelectMenu';
 export { getSelectStyles } from './Select/getSelectStyles';
 export * from './Select/types';
 
+// NewSelect/Combobox
+export * from './Combobox/Combobox';
```

Comment:
Lets not export this yet.

Instead we can export it from `src/unstable.ts`, and import it within grafana core from `@grafana/ui/src/unstable`

Reply:
Ah, thank you, I completely forgot we had that!